src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,16 @@
-export const UserInfo = () => {};
+import './UserInfo.scss';
```

Comment:
Consider importing styles at the top of the file for better readability and consistency. 🎨